odoo/addons/base/views/res_users_identitycheck_views.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+
+        <record id="identity_check_wizard" model="ir.ui.view">
+            <field name="model">res.users.identitycheck</field>
+            <field name="arch" type="xml">
+                <form string="Password Confirmation">
+                    <sheet class="bg-primary">
+                        <h3><strong>Please enter your password to confirm you own this account</strong></h3>
+                        <br/>
+                        <div>
```

Comment:
That's uncommon to add div in standard views... we prefer to let the rendered do its job without trying to add div and custom styling, unless you have strong reasons to do so ? You may play with groups notably, depending on the display you want.

Reply:
Same here, this code doesn't belong to my and I wasn't really thinking about improving it but if you want to, I can.